.cirrus.yml

```diff
@@ -0,0 +1,19 @@
+env:
```

Comment:
> I did not include the ".cirrus.yml" file as it fails to build on "cirrus-ci.com" and as yet I don't know why. It builds and runs perfectly locally, so I need to install another instance of FreeBSD from scratch to see what I am missing.

I see that you have included the file into the PR. Were you able to resolve the issues?

Reply:
I have just installed a virtualbox instance of FreeBSD 12.1... I suspect I installed something on my other system that made this instance of retdet compile correctly, and I didn't realise!

BTW, when my zfs filesystem got corrupted, changes I was making to files were not being updated... I kept saying "that is impossible". Which is why there were so many commits. git status reported files that were different, and they were identical... REALLY frustrating until I found out what was going on.